### ISSUE TITLE
feature request hide legend when unchecking layers

### ISSUE THREAD

**Issue by Jeroen-GroeneBij** [tomchadwin/qgis2web#677]
Hi Tom,

The following is a feature request for the Leaflet webmap. (I am currently using qgis 2.18.20 and qgis2web 2.34.0)
When I use multiple layers and use the Add layers list feature, the webmap shows me the legend of all exported layers. When I uncheck a layer in the webmap, the corresponding legend stays expanded. It would be nice for this specific layer legend to be collapsed when the layer is not selected and be expanded again when the layer is checked/visible.

Is this something you can code? Is this a Leaflet issue?

Thank you.

**Comment by tomchadwin** [comment]
Do you mean the categories/ranges belonging to a layer?

**Comment by Jeroen-GroeneBij** [comment]
Yes, those categories are added as legend through the Add layer list feature.
